prow/plugins/approve/approve.go

```diff
@@ -585,13 +585,24 @@ func addApprovers(approversHandler *approvers.Approvers, approveComments []*comm
 // optionsForRepo gets the plugins.Approve struct that is applicable to the indicated repo.
 func optionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Approve {
 	fullName := fmt.Sprintf("%s/%s", org, repo)
+
+	// First search for repo config
+	for _, i := range config.Approve {
```

Comment:
nit, I'd consider `a` (approve) instead of `i` (index?)

Reply:
you're right, although I went with the more generic `c` for both plugins as it's a config it's iterating through